lms/djangoapps/instructor_task/models.py

```diff
@@ -216,7 +223,7 @@ def _get_utf8_encoded_rows(self, rows):
         compatibility.
         """
         for row in rows:
-            yield [unicode(item).encode('utf-8') for item in row]
+            yield [item.encode('utf-8') for item in row]
```

Comment:
Maybe I've missed a step here.  To me, this looks like a change that won't solve the problem.

This is the error shown in TNL-1447:
```
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/instructor_task/models.py", line 219, in _get_utf8_encoded_rows
    yield [unicode(item).encode('utf-8') for item in row]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe8 in position 0: ordinal not in range(128)
```
It's a DecodeError, meaning a byte string was being decoded to a unicode string. This must have been happening inside the unicode() call, meaning `item` is a byte string.  Removing the call to unicode() won't change the behavior, because the .encode() method call will do precisely the same thing: only a unicode string can be encoded, so Python 2 will implicitly try to convert the byte string to a unicode string, and it will fail in the same way it does now.

Seems like we still haven't got a test case that properly shows the problem.